src/_Samples/CustomerQuery.php

```diff
@@ -14,12 +16,10 @@
   'accessTokenKey'  => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
   'refreshTokenKey' => 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
   'QBORealmID'      => "xxxxxxxxxxxxxxxxxxx",
-  'baseUrl'         => "development",
-  'enableRequestLogging' => true,
-  'loggingTraceLevel' => TraceLevel::Verbose,
+  'baseUrl'         => "development"
 ));
 
-$dataService->setLogLocation(CoreConstants::DEFAULT_LOGGINGLOCATION);
+$dataService->setLogLocation("/Users/hlu2/Desktop/newFolderForLog");
```

Comment:
@abisalehalliprasan can we give some relative path

Reply:
or a temp path

Reply:
As a workaround, we can include the relative path in the dataService object:
```
$dataService = DataService::Configure(array(
  'auth_mode'       => 'oauth2',
  'ClientID'        => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
  'ClientSecret'    => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
  'accessTokenKey'  => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
  'refreshTokenKey' => 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'QBORealmID'      => "xxxxxxxxxxxxxxxxxxx",
  'baseUrl'         => "development",
  'setLogLocation' => 'relative path to save logs'
));
```

But it should be to the `discretion` of the developer to pass the relative path from `config.php`